community/security/src/test/java/org/neo4j/server/security/auth/BasicAuthManagerTest.java

```diff
@@ -59,18 +59,18 @@ public void shouldFindAndAuthenticateUserSuccessfully() throws Throwable
     {
         // Given
         final InMemoryUserRepository users = new InMemoryUserRepository();
-        final User user = new User( "jake", Credential.forPassword( "abc123" ), false );
+            final User user = new User( "jake", "admin", Credential.forPassword( "abc123" ), false );
```

Comment:
We are no actually testing the group, it is just passing something in.

We already have a plan to add multiple groups as a separate card. This is just the minimal first iteration.